waspc/data/Generator/templates/sdk/wasp/client/router/Link.tsx

```diff
@@ -13,7 +13,7 @@ export function Link(
     hash?: string;
   }
   & Routes
-) {
+): React.JSX.Element {
```

Comment:
I haven't done anything fancy with the types in this PR, only explicitly stated what the functions already returned.

That said, I'm still a little fuzzy on these React component return types. It might make sense to return to them one day.